src/pages/Home.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Card from '../components/Card';
+import Form from '../components/Form';
+import Paginator from '../components/Paginator';
+import { defaultCurrentPage, minPageSize, sorts } from '../constants/constants';
+
+const Home = () => {
+  const [searchValue, setSearchValue] = React.useState('');
+
+  const [sortBy, setSortBy] = React.useState(sorts[0]);
+
+  const [pageSize, setPageSize] = React.useState(minPageSize);
+
+  const [totalPages, setTotalPages] = React.useState(0);
+
+  const [currentPage, setCurrentPage] = React.useState(defaultCurrentPage);
+
+  const [isPaginatorBtnClicked, setIsPaginatorBtnClicked] = React.useState(false);
+
+  const [isLoading, setIsLoading] = React.useState(false);
+
+  const [cards, setCards] = React.useState([]);
+
+  return (
+    <>
+      <Form
+        searchValue={searchValue}
+        sortBy={sortBy}
+        pageSize={pageSize}
+        currentPage={currentPage}
+        isLoading={isLoading}
+        setCards={setCards}
+        setIsLoading={setIsLoading}
+        setSearchValue={setSearchValue}
+        setSortBy={setSortBy}
+        setPageSize={setPageSize}
+        setTotalPages={setTotalPages}
+        setCurrentPage={setCurrentPage}
+        setIsPaginatorBtnClicked={setIsPaginatorBtnClicked}
+      />
+      {typeof cards !== 'string' && cards.length > 0 && (
```

Comment:
Why are you checking the type of cards equals string if it expects to be an array?